docs/UserGuide.md

```diff
@@ -115,30 +117,48 @@ Currently, there are 6 types of drinks that can be tracked by tCheck's sales tra
 
 
 #### 1.1 Updating the number of drinks sold : `s-update`
-Updates the number of drinks sold for each drink type as given in the user input.
+Updates the number of drinks sold for each drink type as given in the user input. The previous record will
+be overwritten.
 
-
-Format: `s-update A/NUM B/NUM C/NUM ...`
+Format: `s-update A/NUM [B/NUM] [C/NUM] ...`
 * `A`, `B`, `C` are abbreviations for the drink types.
-* `NUM` refers to the number of drinks sold. It should be a non-negative integer.
+* `NUM` refers to the number of drinks sold. It should be a **non-negative unsigned integer** that is 
+less than or equal to 99999999.
+* You must record the sales of at least one drink item when using this command.
 
-Example:
-* `s-update BSBM/100 BSBBT/120` Updates the sales of Brown Sugar Boba Milk `BSBM` to 100 and
- Brown Sugar Boba Black Tea `BSBBT` to 120.
- 
- <div markdown="span" class="alert alert-primary">
- 
-:information_source: Note about the update: <br>
+Example: Let's say you want to update the sales of Brown Sugar Boba Milk `BSBM` to 100 
+and Brown Sugar Boba Black Tea `BSBBT` to 120 in the Sales Tracker. You can follow these instructions:
 
-The record in tCheck will be overwritten by the input that you provide. <br>
- 
- </div>
+1. Type `s-update BSBM/100 BSBBT/120` into the Command Box.
+2. Press `Enter` to execute the command.
+
+Outcome:
+
+1. The Result Display will show a success message.
```

Comment:
So here would be italics I guess? for result display and tracker

Reply:
Yes, I will update that.